kolibri/plugins/device/assets/src/constants.js

```diff
@@ -14,6 +14,8 @@ export const PageNames = {
   AVAILABLE_CHANNELS_PAGE: 'AVAILABLE_CHANNELS',
   FACILITIES_PAGE: 'FACILITIES_PAGE',
   FACILITIES_TASKS_PAGE: 'FACILITIES_TASKS_PAGE',
+  MANAGE_SYNC_SCHEDULE: 'MANAGE_SYNC_SCHEDULE',
```

Comment:
I see you have defined these in the `kolibri-common` package - I like that idea! That does mean you don't have to repeat them here. For explicitness sake, you could import those SyncPageNames within this file and add them to the `PageNames` constant here using the object spread operator:

```
export const PageNames = {
  MANAGE_CONTENT_PAGE: 'MANAGE_CONTENT_PAGE',
  MANAGE_PERMISSIONS_PAGE: 'MANAGE_PERMISSIONS_PAGE',
  USER_PERMISSIONS_PAGE: 'USER_PERMISSIONS_PAGE',
  DEVICE_INFO_PAGE: 'DEVICE_INFO_PAGE',
  DEVICE_SETTINGS_PAGE: 'DEVICE_SETTINGS_PAGE',
  REARRANGE_CHANNELS: 'REARRANGE_CHANNELS',
  DELETE_CHANNELS: 'DELETE_CHANNELS',
  EXPORT_CHANNELS: 'EXPORT_CHANNELS',
  MANAGE_TASKS: 'MANAGE_TASKS',
  MANAGE_CHANNEL: 'MANAGE_CHANNEL',
  SELECT_CONTENT: 'SELECT_CONTENT',
  NEW_CHANNEL_VERSION_PAGE: 'NEW_CHANNEL_VERSION_PAGE',
  AVAILABLE_CHANNELS_PAGE: 'AVAILABLE_CHANNELS',
  FACILITIES_PAGE: 'FACILITIES_PAGE',
  FACILITIES_TASKS_PAGE: 'FACILITIES_TASKS_PAGE',
  ...SyncPageNames,
};
```

This will add all the keys and values from the `SyncPageNames` object to the `PageNames` object in this file.